README.md

```diff
@@ -238,7 +239,8 @@ The Polymer library uses a BSD-like license that is available [here](./LICENSE.t
 -----------
 
 <a name="release-candidate"></a>
```

Comment:
I was thinking of leaving it in in case folks had it bookmarked, but I don't feel strongly about it ... 🍡